content/en/docs/troubleshooting.md

```diff
@@ -131,10 +131,10 @@ Certificate details
 --------------------------------------------------------------------------------
 Issuer: CN=example.io,O=Notary,L=Seattle,ST=WA,C=US
 Subject: CN=example.io,O=Notary,L=Seattle,ST=WA,C=US
-Valid from: 2023-01-15 07:55:01 +0000 UTC
-Valid to: 2023-01-16 07:55:01 +0000 UTC
+Valid from: 2023-06-22 08:33:39 +0000 UTC
+Valid to: 2023-06-23 08:33:39 +0000 UTC
 IsCA: false
-SHA1 Thumbprint: xxx
+SHA256 Thumbprint: 3ab41b683b24b40abfa87c1fe26c81b917a9ccc1c4bf60f90e2b41deb86f5220
```

Comment:
Would you mind sharing where the thumbprint is from?

Reply:
Yes. In the troubleshooting.md docs, it is mentioned that users can use `notation cert show --type ca --store mytruststore mycertificate.crt` to show the details of the certificate in the trust store. I believe `mytruststore` and `mycertificate.crt` are placeholders that need to be replaced with the actual trust store name and certificate. As @Two-Hearts mentioned, "`notation cert show` command currently displays SHA256 and not SHA1 thumbprints." To update this, I ran the command:

```ruby
notation cert show --type ca --store wabbit-networks.io wabbit-networks.io.crt
```
In the above command, I replaced `mytruststore` to `wabbit-networks.io` and mycertificate.crt` to `wabbit-networks.io.crt` 

This is the complete output:

```ruby
Certificate details
--------------------------------------------------------------------------------
Issuer: CN=wabbit-networks.io,O=Notary,L=Seattle,ST=WA,C=US
Subject: CN=wabbit-networks.io,O=Notary,L=Seattle,ST=WA,C=US
Valid from: 2023-06-22 08:33:39 +0000 UTC
Valid to: 2023-06-23 08:33:39 +0000 UTC
IsCA: false
SHA256 Thumbprint: 3ab41b683b24b40abfa87c1fe26c81b917a9ccc1c4bf60f90e2b41deb86f5220
```